examples/asrc_demo/README.md

```diff
@@ -0,0 +1,53 @@
+# ASRC demo application
+
+This is the ASRC demo example design firmware.  See the full documentation for more information on configuring, modifying, building, and running the firmware.
+
+## Supported Hardware
+
+This example is supported on the XK_VOICE_L71 board.
+
+
+## Building the Firmware
+
+Run the following commands in the root folder to build the firmware.
+
+On Linux and Mac run:
+
+    cmake -B build -DCMAKE_TOOLCHAIN_FILE=xmos_cmake_toolchain/xs3a.cmake
+    cd build
+
+    make example_ffva_asrc_otg
+
+On Windows run:
+
+    cmake -G "NMake Makefiles" -B build -DCMAKE_TOOLCHAIN_FILE=xmos_cmake_toolchain/xs3a.cmake
+    cd build
+
+    nmake example_ffva_asrc_otg
+
+From the build folder, flash the device with the appropriate command to the desired configuration:
+
+On Linux and Mac run:
+
+    make flash_app_example_ffva_asrc_otg
+
+On Windows run:
+
+    nmake flash_app_example_ffva_asrc_otg
```

Comment:
have you confirmed nmake works OK? We have move to Ninja for many examples now
https://github.com/xmos/sln_voice/blob/develop/examples/mic_aggregator/README.rst

Reply:
I haven't checked for Windows. Just copied this from the ffva example which was the starting point for this. I'll test it on a Windows machine.